File: src/Model/Werewolf.java
package Model;

import Technical.PowerEnum;

public class Werewolf extends GameCharacter {

    public Werewolf() {
        super("Werewolf",
                "The werewolf can select a victim during the night to eat and eliminate from the game.",
                true,
                PowerEnum.Constant,
                "../../img/Thiercelieux.png");
    }
}

File: app/src/main/java/Model/Werewolf.kt
package Model

class Werewolf(order: Int)
    : Character("The werewolf can select a victim during the night to eat and eliminate from the game."
    , true
    , PowerState.PERMANENT
    , true
    , order
    , 5) {

    override fun action(): String {
        return "The $className choose a victim to eat."
    }

    override fun clone() = Werewolf(this.order)
}